runtime/src/lib.rs

```diff
@@ -366,7 +413,7 @@ parameter_types! {
 }
 
 impl pallet_finality_tracker::Trait for Runtime {
-    type OnFinalizationStalled = Grandpa;
+    type OnFinalizationStalled = ();
```

Comment:
What behavior do we loose or gain with this change?

Reply:
Subsrate code for the finality tracker trait:
```
	/// Something which can be notified when the timestamp is set. Set this to `()`
	/// if not needed.
	type OnFinalizationStalled: OnFinalizationStalled<Self::BlockNumber>;
```
The Substrate template node runtime sets this to ().
Polkadot runtime doesn't contain the finality tracker at all.